### ISSUE TITLE
[Slider] Color specs update

### ISSUE THREAD

**Issue by aweell** [dxc-technology/halstack-style-guide#311]
## Story description

- [ ] Update component documentation page with corrected color specs

## React

```
fontColor: globalTokens.black
fontColorOnDark: globalTokens.white
thumbBackgroundColor: globalTokens.mediumBlue
thumbBackgroundColorOnDark: globalTokens.white
draggedThumbBackgroundColor: globalTokens.mediumBlue
draggedThumbBackgroundColorOnDark: "#0095FF"
dotsBackgroundColor: globalTokens.mediumBlue
dotsBackgroundColorOnDark: globalTokens.white
trackLineColor: globalTokens.mediumBlue
trackLineOnDark: globalTokens.white
totalLineColor: globalTokens.softBlue
totalLineOnDark: globalTokens.white
disabledThumbBackgroundColor: globalTokens.softBlue
disabledThumbBackgroundColorOnDark: "#575757"
disabledDotsBackgroundColor: globalTokens.softBlue
disabledDotsBackgroundColorOnDark: "#575757"
disabledTrackLineColor: globalTokens.softBlue
disabledTrackLineOnDark: "#575757"
disabledTotalLineColor: globalTokens.softBlue
disabledTotalLineColorOnDark: "#575757"
focusColor: globalTokens.blue
focusColorOnDark: "#1682FF"  
```

## Angular

```
"--slider-thumbBackgroundColor": globalTokens.mediumBlue
"--slider-dotsBackgroundColor": globalTokens.mediumBlue
"--slider-trackLine": globalTokens.mediumBlue
"--slider-totalLine": globalTokens.softBlue
"--slider-disabledThumbBackgroundColor": globalTokens.softBlue
"--slider-disabledDotsBackgroundColor": globalTokens.softBlue
"--slider-disabledTrackLine": globalTokens.softBlue
"--slider-disabledtotalLine": globalTokens.softBlue
"--slider-focusColor": globalTokens.blue
```

### On-Light

|   Component token                                      |   Element                              |   Core token                         |   Value             |
| ----------------------------------------- | -------------------------- | -------------------------- | -------------- |
|  `fontColor`                                                  |   Floor/Ceil label                   |   `Hal-Black`                       |   `#000000`   |
|  `thumbBackgroundColor`                         |   Thumb                                 |   `Hal-Blue-S-35`              |   `#0067b3`   |
|  `activeThumbBackgroundColor`              |   Thumb:active                      |   `Hal-Blue-D-20`              |   `#003c66`   |
|  `disabledThumbBackgroundColor`         |   Thumb:disabled                  |   `Hal-Blue-L-95`               |   `#e6f4ff`      |
|  `tickMarkBackgroundColor`                     |   Tick Mark                             |   `Hal-Blue-S-35`              |   `#0067b3`   |
|  `disabledTickMarkBackgroundColor`      |   Tick Mark:disabled             |   `Hal-Blue-L-95`               |   `#e6f4ff`     |
|  `trackLineColor`                                         |   Track line                             |   `Hal-Blue-S-35`              |   `#0067b3`  |
|  `disabledTrackLineColor`                         |    Track line:disabled             |   `Hal-Blue-L-95`               |   `#e6f4ff`     |
|  `totalLineColor`                                         |    Total line                             |   `Hal-Grey-L-75`                |   `#bfbfbf`    |
|  `disabledTotalLineColor`                          |    Total line:disabled             |   `Hal-Grey-L-95`                |   `#f2f2f2`     |
|  `focusColor`                                               |   Focus outline                     |   `Hal-Blue-S-35`                |   `#0067b3`  |

### On-Dark

|   Component token                                                 |   Element                   |   Core token               |   Value      |
| ----------------------------------------------- | --------------------------- | -------------------------- | ------------ |
|  `fontColorOnDark`                                                |  Floor/Ceil label             |   `Hal-White`                  |   `#ffffff`         |
|  `thumbBackgroundColorOnDark`                       |  Thumb                           |   `Hal-Blue-L-50`          |   `#0095ff`     |
|  `activeThumbBackgroundColorOnDark`            |  Thumb:active                |   `Hal-Blue-S-35`         |   `#0067b3`   |
|  `disabledThumbBackgroundColorOnDark`       |  Thumb:disabled            |   `Hal-Blue-D-20`         |   `#003c66`  |
|  `tickMarkBackgroundColorOnDark`                   |   Tick Mark                     |    `Hal-Blue-L-50`          |   `#0095ff`     |
|  `disabledTickMarkBackgroundColorOnDark`    |   Tick Mark:disabled     |    `Hal-Blue-D-20`         |   `#003c66`  |
|  `trackLineOnDark`                                                |   Track line                     |   `Hal-Blue-L-50`          |   `#0095ff`     |
|  `disabledTrackLineColorOnDark`                        |   Track line:disabled     |   `Hal-Blue-D-20`         |   `#003c66`  |
|  `totalLineOnDark`                                                 |   Total line                      |   `Hal-Grey-L-75`           |   `#bfbfbf`    |
|  `disabledTotalLineColorOnDark`                         | Total line:disabled        |  `Hal-Grey-S-40`           |   `#666666`  |
|  `focusColorOnDark`                                             |  Focus outline                |   `Hal-Blue-L-50`          |   `#0095ff`     |

**Comment by mnnaveenk** [comment]
## React

fontColor: globalTokens.black
fontColorOnDark: globalTokens.white
thumbBackgroundColor: globalTokens.mediumBlue
thumbBackgroundColorOnDark: globalTokens.white
draggedThumbBackgroundColor: globalTokens.mediumBlue
draggedThumbBackgroundColorOnDark: "#0095FF"
dotsBackgroundColor: globalTokens.mediumBlue
dotsBackgroundColorOnDark: globalTokens.white
trackLineColor: globalTokens.mediumBlue
trackLineOnDark: globalTokens.white
totalLineColor: globalTokens.softBlue
totalLineOnDark: globalTokens.white
disabledThumbBackgroundColor: globalTokens.softBlue
disabledThumbBackgroundColorOnDark: "#575757"
disabledDotsBackgroundColor: globalTokens.softBlue
disabledDotsBackgroundColorOnDark: "#575757"
disabledTrackLineColor: globalTokens.softBlue
disabledTrackLineOnDark: "#575757"
disabledTotalLineColor: globalTokens.softBlue
disabledTotalLineColorOnDark: "#575757"
focusColor: globalTokens.blue
focusColorOnDark: "#1682FF"  

## Angular

"--slider-thumbBackgroundColor": globalTokens.mediumBlue
"--slider-dotsBackgroundColor": globalTokens.mediumBlue
"--slider-trackLine": globalTokens.mediumBlue
"--slider-totalLine": globalTokens.softBlue
"--slider-disabledThumbBackgroundColor": globalTokens.softBlue
"--slider-disabledDotsBackgroundColor": globalTokens.softBlue
"--slider-disabledTrackLine": globalTokens.softBlue
"--slider-disabledtotalLine": globalTokens.softBlue
"--slider-focusColor": globalTokens.blue

## Slider color specs table


#Dark

|   Component token                       |   Element                   |   Core token               |   Value      |
| --------------------------------------- | --------------------------- | -------------------------- | ------------ |
|  `fontColorOnDark`                      |                             |   `Hal-White`              |   `#ffffff`  |
|  `thumbBackgroundColorOnDark`           |                             |   `Hal-White`              |   `#ffffff`  |
|  `draggedThumbBackgroundColorOnDark`    |                             |   `Hal-Blue-L-50`          |   `#0095FF`  |
|  `dotsBackgroundColorOnDark`            |                             |   `Hal-White`              |   `#ffffff`  |
|  `trackLineOnDark`                      |                             |   `Hal-White`              |   `#ffffff`  |
|  `totalLineOnDark`                      |                             |   `Hal-White`              |   `#ffffff`  |
|  `disabledThumbBackgroundColorOnDark`   |                             |   -                        |   `#575757`  |
|  `disabledDotsBackgroundColorOnDark`    |                             |   -                        |   `#575757`  |
|  `disabledTrackLineOnDark`              |                             |   -                        |   `#575757`  |
|  `disabledTotalLineColorOnDark`         |                             |   -                        |   `#575757`  |
|  `focusColorOnDark`                     |                             |   -                        |   `#1682FF`  |


#Light

|   Component token                       |   Element                   |   Core token               |   Value      |
| --------------------------------------- | --------------------------- | -------------------------- | ------------ |
|  `fontColor`                            |                             |   `Hal-Black`              |   `#000000`  |
|  `thumbBackgroundColor`                 |                             |   `Hal-Blue-S-35`          |   `#0067b3`  |
|  `draggedThumbBackgroundColor`          |                             |   `Hal-Blue-S-35`          |   `#0067b3`  |
|  `dotsBackgroundColor`                  |                             |   `Hal-Blue-S-35`          |   `#0067b3`  |
|  `trackLineColor`                       |                             |   `Hal-Blue-S-35`          |   `#0067b3`  |
|  `trackLine`                            |                             |   `Hal-Blue-S-35`          |   `#0067b3`  |
|  `totalLineColor`                       |                             |   `globalTokens.softBlue`  |   `#b1cee6`  |
|  `totalLine`                            |                             |   `globalTokens.softBlue`  |   `#b1cee6`  |
|  `disabledThumbBackgroundColor`         |                             |   `globalTokens.softBlue`  |   `#b1cee6`  |
|  `disabledDotsBackgroundColor`          |                             |   `globalTokens.softBlue`  |   `#b1cee6`  |
|  `disabledTrackLine`                    |                             |   `globalTokens.softBlue`  |   `#b1cee6`  |
|  `disabledTrackLineColor`               |                             |   `globalTokens.softBlue`  |   `#b1cee6`  |
|  `disabledtotalLine`                    |                             |   `globalTokens.softBlue`  |   `#b1cee6`  |
|  `disabledTotalLineColor`               |                             |   `globalTokens.softBlue`  |   `#b1cee6`  |
|  `focusColor`                           |                             |   `globalTokens.blue`      |   `#005FCC`  |

**Comment by GomezIvann** [comment]
Changes made on the Slider advanced tokens:

- [ ] `totalLineThickness` has a new value: `2px`.
- [ ] `disabledDotsVerticalPosition` has a new value: `42%`.

**Comment by aweell** [comment]
Closed via #416 and #423
